packs/linux/actions/get_live_hosts.yaml

```diff
@@ -0,0 +1,11 @@
+name: get_live_hosts
+runner_type: run-python
+description: List all hosts on a given Network
+enabled: true
+entry_point: get_live_hosts.py
+parameters:
+  network:
+    type: string
+    description: "Network to query live hosts "
```

Comment:
yeah, You are right, I have changed the name from network to subnet and remove the main entry point.